src/popup/components/TransactionInfoDetailsParty.vue

```diff
@@ -11,7 +11,7 @@
     >
       <Truncate
         :right="isRecipient"
-        :str="txParty.name || txParty.label"
+        :str="txParty.name || txParty.label || 'Fellow Superhero'"
```

Comment:
Fellow Superhero -> move to translate

Reply:
It's still to be agreed what should be presented here